tox.ini

```diff
@@ -9,7 +9,9 @@ changedir = {toxinidir}/raiden
 deps =
     -rrequirements.txt
     pdbpp
+    pep8
```

Comment:
flake8 is a wrapper around pep8/pyflakes/mccabe, I decided to start with the simpler option, in other words, these changes are required but not sufficient to use flake8 and I did *not* look at flake8 at this moment.